docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+## Glossary
```

Comment:
I have updated the glossary part at https://github.com/Cloud-CV/EvalAI/wiki/Glossary. Please copy the markdown code.